### DISCUSSION TITLE
Can I use a PS3 controller wirelessly?

### DISCUSSION THREAD

**Discussion by LAPIII** [AntiMicroX/antimicrox#483] [Q&A]
?

**Comment by pktiuk** [comment]
Yes, you can, just connect it via Bluetooth and it should be detected by app.

**Comment by LAPIII** [comment]
My OS is Windows 10 and my PS3 controller isn't showing up in AntiMicroX, (https://postimg.cc/zLwTGfFC). In Settings, it shows up under other devices (https://postimg.cc/fVfb3S1R) and I'm not sure what to do. I clicked 'Add Bluetooth or other device' and then 'Bluetooth' (https://postimg.cc/WhPjMbGF), but the device isn't listed.

**Reply by LAPIII** [reply]
Instead I connected my Xbox controller.

**Reply by pktiuk** [reply]
Have you tested your gamepad with other apps like `qjoypad`, `jstest-gtk` ? Does it work properly with them?
